packages/tokens/package.json

```diff
@@ -20,9 +20,13 @@
     },
     "type": "module",
     "exports": {
-        "./dist/": {
-            "import": "./dist/",
-            "require": "./dist/"
+        "./dist/tokens.css": {
+            "import": "./dist/tokens.css",
+            "require": "./dist/tokens.css"
+        },
+        "./dist/fonts.css": {
+            "import": "./dist/fonts.css",
+            "require": "./dist/fonts.css"
```

Comment:
This really look overkill, is it required for your fix?

Reply:
not necessarily.. I was just trying some suggestions I found online. Is there something you'd recommend instead? this is not something I do often.

Reply:
It's just that it can be error prone to have to specify manually each of your file in the package, instead of just specifying the folder.